rubygem-smart_proxy_remote_execution_ssh_core/rubygem-smart_proxy_remote_execution_ssh_core.spec

```diff
@@ -0,0 +1,77 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%if 0%{?scl:1}
+    %global bin_dir %{_root_bindir}
+%else
+    %global bin_dir %{_bindir}
+%endif
+
+%global gem_name smart_proxy_remote_execution_ssh_core
+
+Summary: Smart Proxy Remote Execution SSH Core
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.1
+Release: 1%{?foremandist}%{?dist}
+Group: Development/Libraries
+License: GPLv3
+URL: http://github.com/theforeman/smart_proxy_remote_execution_ssh
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix}rubygem(net-scp)
+Requires: %{?scl_prefix}rubygem(net-ssh) <= 2.9.4
```

Comment:
Ah I see, because we used to require older ruby.  Newer net-ssh should be ok in SCL: https://github.com/theforeman/smart_proxy_remote_execution_ssh/pull/25